example/package.json

```diff
@@ -4,11 +4,19 @@
   "description": "Showing how serverless-plugin-ignitor can be used",
   "scripts": {
     "deploy": "sls deploy",
-    "local:ignitor": "sls invoke local -f hello -p fixtures/ignitor.json",
+    "local:hello": "sls invoke local --function hello",
```

Comment:
Change `sls` to `serverless`. I believe `sls` only works for globally installed serverless

Reply:
Good point